src/Compilers/CSharp/Portable/Syntax/CSharpSyntaxTree.cs

```diff
@@ -491,8 +554,9 @@ private SyntaxTree WithChanges(SourceText newText, IReadOnlyList<TextChangeRange
                 Options,
                 compilationUnit,
                 parser.Directives,
+#pragma warning disable CS0618
```

Comment:
We shouldn't be using obsolete methods in real code.

Reply:
See comment at https://github.com/dotnet/roslyn/pull/44331#discussion_r460245706; this is preserving the behavior that @agocke wanted to preserve.